npm-packages/meteor-installer/package.json

```diff
@@ -9,16 +9,19 @@
   "author": "zodern",
   "license": "MIT",
   "dependencies": {
-    "7zip-bin": "^5.0.3",
-    "cli-progress": "^3.5.0",
-    "node-7z": "^2.0.5",
-    "node-downloader-helper": "^1.0.11",
+    "7zip-bin": "^5.2.0",
+    "cli-progress": "^3.11.1",
+    "node-7z": "^2.1.2",
+    "node-downloader-helper": "^1.0.19",
     "rimraf": "^3.0.2",
-    "semver": "^7.3.5",
-    "tar": "^6.1.0",
-    "tmp": "^0.1.0"
+    "semver": "^7.3.7",
+    "tar": "^6.1.11",
+    "tmp": "^0.2.1"
   },
   "bin": {
     "meteor-installer": "cli.js"
+  },
+  "engines": {
+    "node": "<=14.19.3"
```

Comment:
Would using something like `<15` work? That way it would include any new 14.x releases.

Reply:
Now I'm using "node": "<=14.x", and "npm": "<=6.x". This is just a warning, and it will not generate an error.